scripts/target_lib.lua

```diff
@@ -44,6 +44,7 @@ local default_options = {
 	aliases = {},
 	manifest_aliases = {},
 	packages = {},
+	featureset = "standard",
```

Comment:
Hmm, I wonder if "featureset" is really a good name for this - might get confusing, as we already have "features". And by itself, it categorizes devices rather than features - maybe something like "device class" (or just "class" in the target definitions) would be more appropriate?